public_html/wp-content/plugins/wordcamp-payments/includes/reimbursement-request.php

```diff
@@ -250,7 +252,7 @@ function enqueue_assets() {
  * requests and to submit notes, but they should only be able to change the form fields if the post hasn't been
  * submitted yet, or if we've asked for more information.
  *
- * @param \WP_Post $post
+ * @param WP_Post $post
```

Comment:
Since there's a lot of PHPCS fixes, I think it'd be ideal for them to be in a separate commit, to avoid diff noise, but it's probably not worth breaking the commits apart now that they're already pushed.

Reply:
Good point